src/contexts/RainbowProvider/index.tsx

```diff
@@ -51,10 +52,17 @@ const RainbowProvider = props => {
 const Web3ContextProvider = props => {
   const { connector: activeConnector, address, isConnected, chainId } = useAccount()
   const { data: client } = useConnectorClient<Config>({ chainId })
-
   const { openConnectModal } = useConnectModal()
   const { disconnect } = useDisconnect()
 
+  useEffect(() => {
+    if (isConnected && activeConnector?.name) {
+      sendGAEvent("wallet_connected", {
```

Comment:
Based on the code above, it seems "event" is missing here.
```
sendGAEvent("event", "wallet_connected", {
  ...
}
```

Reply:
Good catch, thanks! Updated.